src/io_source.rs

```diff
@@ -198,3 +206,87 @@ where
         fmt::Debug::fmt(&self.inner, f)
     }
 }
+
+/// Used to associate an `IoSource` with a `sys::Selector`.
+#[derive(Debug)]
+#[cfg(debug_assertions)]
```

Comment:
nit: It's not order dependent, but could the `cfg` come the `derive`? I think it's more more important to see that a type is conditionally compiled than the attributes it derives *when* it's included.

Reply:
Done.